ceci/stage.py

```diff
@@ -72,51 +82,115 @@ def __init__(self, args, comm=None):
         ----------
         args: dict or namespace
             Specification of input and output paths and any missing config options
-        comm: MPI communicator
```

Comment:
It looks like this is still a keyword arg - any reason not to document it here?

Reply:
Ok, fixed.